R/rep.R

```diff
@@ -36,6 +36,8 @@
 #'   For `vec_rep_each()`, an integer vector of the number of times to repeat
 #'   each element of `x`. `times` will be [recycled][vector_recycling_rules] to
 #'   the size of `x`.
+#' @inheritParams rlang::args_error_context
```

Comment:
I thought we always put `@inheritParams` tags above all other `@param` tags (i.e. `@inheritParams` would be the first tag listed)